pkg/kp/kv.go

```diff
@@ -50,6 +52,32 @@ func (err KVError) Error() string {
 	return fmt.Sprintf("%s failed for path %s", err.Op, err.Key)
 }
 
+func (c consulStore) Put(key, value string) (time.Duration, error) {
+	value = appendTimeStamp(value)
```

Comment:
Whatever value we write into the KV store it should be in a standard data representation that we have libraries for marshaling and unmarshaling. My strong suggestion is to use JSON for this purpose.